corehq/apps/data_interfaces/templates/data_interfaces/partials/update_rule_run_history.html

```diff
@@ -0,0 +1,37 @@
+{% load hq_shared_tags %}
+{% load i18n %}
+
+<div id="ko-tabs-rule-run-history" class="ko-template">
+  <div class="spacer"></div>
+
+  <div class="panel panel-default">
+    <div class="panel-body">
+      <table class="table table-striped table-hover">
+        <thead>
+        <tr>
+          <th>{% trans "Case Type" %}</th>
+          <th>{% trans "Started On" %}</th>
+          <th>{% trans "Finished On" %}</th>
+          <th>{% trans "# of Cases Updated" %}</th>
+          <th>{% trans "# of Cases Closed" %}</th>
+        </tr>
+        </thead>
+        <tbody data-bind="foreach: paginatedRuleRuns">
+          <tr>
```

Comment:
Should this table include status?

Reply:
That would make sense.

Reply:
See d3c670b8fb0f6bd4885a846cb76b5361c172dd7d